runtime/stream.c

```diff
@@ -480,7 +480,13 @@ strmNextFile(strm_t *pThis)
 	 * make sure their iMaxFiles is large enough. But it is well-desired for other
 	 * use cases, e.g. a circular output log file. -- rgerhards, 2008-01-10
 	 */
-	pThis->iCurrFNum = (pThis->iCurrFNum + 1) % pThis->iMaxFiles;
+	if((pThis->iCurrFNum + 1) == 0 || pThis->iMaxFiles == 0) {
```

Comment:
besides extra check after the assertion, this is still a valid case. I will remove the extra